Makefile

```diff
@@ -38,18 +38,24 @@ scripts/terraform-outputs-gce.sh: check-env-vars gce/${DEPLOY_ENV}.tfstate
 prepare-provision-aws: set-aws manifests/templates/outputs/terraform-outputs-aws.yml scripts/terraform-outputs-aws.sh prepare-provision
 prepare-provision-gce: set-gce manifests/templates/outputs/terraform-outputs-gce.yml scripts/terraform-outputs-gce.sh prepare-provision
 prepare-provision: check-env-vars bastion
+	scp -r -oStrictHostKeyChecking=no cf-manifest/deployments \
+	    ubuntu@${bastion}:
```

Comment:
This would probably be better uploaded to a `cf-manifest` directory on the bastion.  That way the terraform outputs, secrets etc can be uploaded to the locations under there that the existing `build_manifest.sh` expects.